src/site/stages/build/process-cms-exports/transformers/taxonomy_term-health_care_service_taxonomy.js

```diff
@@ -0,0 +1,35 @@
+const { getDrupalValue } = require('./helpers');
+
+const transform = entity => ({
+  entity: {
+    entityType: 'taxonomy_term',
+    entityBundle: 'health_care_service_taxonomy',
+    name: getDrupalValue(entity.name),
+    description: {
+      processed: getDrupalValue(entity.description),
```

Comment:
Should this be `getWysiwygString` (or whatever we've called it)?

Reply:
Good catch!  I'll update that.